examples/node/lib/rest/api/v2010/account.ts

```diff
@@ -20,14 +20,34 @@ const deserialize = require("../../../base/deserialize");
 const serialize = require("../../../base/serialize");
 import { CallListInstance } from "./account/call";
 
+type TestStatus =
```

Comment:
Changes look good, but why aren't we declaring enums the [traditional way](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)?

Reply:
Matching existing functionality.